libs/design/src/atoms/form/select/select/select.component.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../../scss/daff-util';
+
+.daff-select {
+  display: block;
+  position: relative;
+  font-size: 1rem;
+
+  select {
+    display: block;
+    width: 100%;
+    background: transparent;
+    border: 1px solid daff-color($daff-gray, 300);
+    border-radius: 3px;
+    box-shadow: none;
+    padding: 10px 15px;
+    margin: 0;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    min-width: 80px;
+    background-clip: padding-box;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+
+    &:focus {
+      border: 1px solid $black;
+      box-shadow: none;
+      outline: 0;
+    }
+
+    // removes dotted border on in FF
+    &:-moz-focusring {
+      color: transparent;
+      text-shadow: 0 0 0 $black;
+    }
+  }
+
+  &__icon {
+    display: block;
+    position: absolute;
+    top: 15px;
+    right: 15px;
+    z-index: -1;
```

Comment:
This is hacky. I don't think you should do a transparent background, either. See: https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

Reply:
fixed fd2c96e